index.js

```diff
@@ -6,6 +6,7 @@ module.exports = `type Questionnaire {
     legalBasis: LegalBasis
     navigation: Boolean
     surveyId: String
+    createdAt: String
```

Comment:
I wonder if we should use a custom Date type for this?  e.g. https://github.com/excitement-engineer/graphql-iso-date

It might not be worth the effort, but I'm going to ask on GraphQL slack if it's advised.

Reply:
Have asked around, and people said they used a custom Date scalar type. Perhaps we should do the same